CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 
 # 12.0.0-alpha.9 (Unreleased)
 
+#### :bug: Bug fix
```

Comment:
Shouldn't this rather go under "New Feature" or maybe "Polish"?

Reply:
Polish make sense, I'll amend.